src/web3-contracts.js

```diff
@@ -353,8 +355,14 @@ export function useAntStaked() {
       Web3EthContract.setProvider('wss://mainnet.eth.aragon.network/ws')
       const ANT_ADDR = getKnownContract('TOKEN_ANT')[0]
       const ant = new Web3EthContract(tokenBalanceOfAbi, ANT_ADDR)
-      const antStaked = await ant.methods.balanceOf(NETWORK_AGENT_ADDR).call()
-      setAntStaked(String(antStaked))
+      const antStakedInAgent = bigNum(
+        await ant.methods.balanceOf(NETWORK_AGENT_ADDR).call()
```

Comment:
I missed the introduction of `Web3EthContract`, is there a reason why we can’t use `useContract()` (= ethers.js) here?  :thinking:

Reply:
There isn't; it must've been overlooked when we added in the stats module.

Reply:
Cool, let’s remove it soon!

I think we could try https://bundlewatch.io/ using [this action](https://github.com/jackyef/bundlewatch-gh-action).